SourceCpp/Diffusion.cpp

```diff
@@ -117,11 +117,14 @@ PeleC::getMOLSrcTerm(
     // }
     // const int* domain_lo = geom.Domain().loVect();
     // const int* domain_hi = geom.Domain().hiVect();
-
     for (amrex::MFIter mfi(MOLSrcTerm, amrex::TilingIfNotGPU()); mfi.isValid();
          ++mfi) {
       const amrex::Box vbox = mfi.tilebox();
-      int ng = S.nGrow();
+#ifdef PELEC_USE_EB
```

Comment:
Ok I see what you mean... changing this always causes us headaches. We will have to hope the CI and nightlys catch all the potential memory issues that might arise from this. I am a little confused as to why this is necessary... could you elaborate?

Reply:
For the spray stuff, I Sborder could potentially have as much as 6 ghost cells. However, the diffusion operations are completely independent of the spray operations. So the number of ghost cells should not depend on the size of Sborder. It causes problems trying to solve the derivatives 5-6 ghost cells out if I don't change ng.